install/kubernetes/cilium/values.yaml

```diff
@@ -24,11 +23,9 @@ debug:
   # - datapath
   # - policy
   verbose: ~
-
```

Comment:
All these line removals are not present in the `values.yaml.tmpl`. Before this change, `values.yaml` and `values.yaml.tmpl` were virtually identical. Now they have different formatting. 

Can we also format the `values.yaml.tmpl` using the same formatter as the `values.yaml`? That will make it easier to use tools like `diff` to observe what has been changed by the templating, which is useful for troubleshooting.

Reply:
Yeah `yq` removed the blank lines. I can see if we can run `yq` on `values.yaml.tmpl` but I'm not sure it's always valid YAML before being templated, so I would need to check.